assets/css/base.scss

```
@@ -77,10 +77,26 @@ $bold: 700;
   margin-right: 0.35rem;
 }
 
+// FUNCTIONS
+
+$browser-context: 16;
+
+@function rem($pixels, $context: $browser-context) {
```

The function works fine. Only thing: I think the name should be a tad more concrete. Something like `pxToRem`. To make it clear for other contributors that it won’t convert, say, `ch` or percentage units.

```suggestion
@function pxToRem($pixels, $context: $browser-context) {
```